### PR TITLE
Added Region Support [SVCINT-1769]

### PR DESCRIPTION
**Pull request by tedre191**
Added the ability to change the Coveo subdomain to support different regions.

### CODE REVIEW

**Review by pathiery — APPROVED** [review]

**Review by mikegron — APPROVED** [review]
lgtm

**Review by jbvouma — APPROVED** [review]

**Review by erocheleau — APPROVED** [review]
Maybe just string revision and an example for the `insightPanelFieldsAllowlist` to make sure no confusion for example do I add spaces or no spaces? API names or field names etc.

File: zendesk-package/translations/en.json [hunk 1]
```diff
@@ -5,9 +5,21 @@
...
     "installation_instructions": "Contact a Coveo representative at coveoforzendesk@coveo.com to get started with your Coveo for Zendesk installation.",
     "parameters": {
-      "organizationId" : { "label": "ID of your Coveo Cloud Org" },
-      "insightPanelFieldsAllowlist"     : { "label": "Fields Allowlist (optional)", "helpText": "Inclusive list of fields to use for context separated by semicolon. All fields sent by default." },
-      "insightPanelFieldsBlocklist"    : { "label": "Fields Blocklist (optional)", "helpText": "List of fields to exclude from the context separated by semicolon. All fields sent by default." }
+      "organizationId": {
+        "label": "ID of your Coveo Cloud Org"
```
Comment (mdas1) [inline zendesk-package/translations/en.json:9]:
```suggestion
        "label": "Coveo Cloud Organization ID"
```

File: zendesk-package/translations/en.json [hunk 2]
```diff
@@ -5,9 +5,21 @@
...
+      },
+      "organizationSubdomain": {
+        "label": "Subdomain of your Coveo Cloud Org",
```
Comment (mdas1) [inline zendesk-package/translations/en.json:12]:
```suggestion
        "label": "Coveo Cloud Organization Subdomain",
```

File: zendesk-package/translations/en.json [hunk 3]
```diff
@@ -5,9 +5,21 @@
...
+        "helpText": "Default subdomain is 'search'. For the EU or AU platforms, use 'search-eu' or 'search-au'."
```
Comment (mdas1) [inline zendesk-package/translations/en.json:13]:
```suggestion
        "helpText": "The default subdomain is 'search'. For the EU or AU platforms, use 'search-eu' or 'search-au'."
```

File: zendesk-package/translations/en.json [hunk 4]
```diff
@@ -5,9 +5,21 @@
...
+      },
+      "insightPanelFieldsAllowlist": {
+        "label": "Fields Allowlist (optional)",
+        "helpText": "Inclusive list of fields to use for context separated by semicolon. All fields sent by default."
```
Comment (mdas1) [inline zendesk-package/translations/en.json:17]:
```suggestion
        "helpText": "Inclusive list of fields to use for context separated by semicolons. All fields sent by default."
```

File: zendesk-package/translations/en.json [hunk 5]
```diff
@@ -5,9 +5,21 @@
...
+      },
+      "insightPanelFieldsBlocklist": {
+        "label": "Fields Blocklist (optional)",
+        "helpText": "List of fields to exclude from the context separated by semicolon. All fields sent by default."
```
Comment (mdas1) [inline zendesk-package/translations/en.json:21]:
```suggestion
        "helpText": "List of fields to exclude from context separated by semicolons. All fields sent by default."
```

**Pull request merged**